src/ExecutionHelper.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright 2018 ThoughtWorks, Inc.
+//
+// This file is part of Gauge-CSharp.
+//
+// Gauge-CSharp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Gauge-CSharp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Gauge-CSharp.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Gauge.CSharp.Core;
+using Gauge.CSharp.Lib;
+using Gauge.Dotnet.Models;
+using Gauge.Dotnet.Strategy;
+using Gauge.Dotnet.Wrappers;
+using Gauge.Messages;
+using Google.Protobuf;
+
+namespace Gauge.Dotnet
+{
+    public class ExecutionHelper : IExecutionHelper
```

Comment:
This is not a helper. In general helper is a small, we should aim to have behaviour encapsulated in a proper class. 

recommendation: rename the class and interface to something more appropriate